### PR TITLE
Creating Book Model

### PR DESCRIPTION
**Pull request by SebasWolox**
## Summary
Adding migration and model

## Trello Card
https://trello.com/c/qORokMBD/19-crear-modelo-de-book

### CODE REVIEW

**Comment by SebasWolox** [comment]
@mtejedorwolox ya resolví los nuevos conflictos, hace falta su revisión de este branch para poder realizar el merge

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Book < ApplicationRecord
+  validates :genre, :autor, :image, :title, :editor, :year, presence: true
```
Comment (mtejedorwolox) [inline app/models/book.rb]: validate presence of genre, author, title, publisher and year only

File: db/migrate/20181010192630_create_books.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre
+      t.string :autor
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: author

File: db/migrate/20181010192630_create_books.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      t.string :image
+      t.string :title
+      t.string :editor
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: publisher

File: db/migrate/20181010192630_create_books.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      t.string :year
+
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: add description

Comment (SebasWolox) [reply db/migrate/20181010192630_create_books.rb]: @mtejedorwolox are we going to use that attribute as string or text type?

File: db/migrate/20181010192630_create_books.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: null: false

File: db/migrate/20181010192630_create_books.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :author
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: same

File: db/migrate/20181010192630_create_books.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :image
+      t.string :title
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: same

File: db/migrate/20181010192630_create_books.rb [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :publisher
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: same

File: db/migrate/20181010192630_create_books.rb [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :year
```
Comment (mtejedorwolox) [inline db/migrate/20181010192630_create_books.rb]: same

**Pull request merged**